src/strings/en-us.json

```diff
@@ -164,8 +164,8 @@
     "DetectingDevices": "Detecting devices",
     "DeviceAccessHelp": "This only applies to devices that can be uniquely identified and will not prevent browser access. Filtering user device access will prevent them from using new devices until they've been approved here.",
     "DirectPlaying": "Direct playing",
-    "DirectStreamHelp1": "The media is compatible with the device regarding resolution and media type (H.264, AC3, etc), but in an incompatible file container (mkv, avi, wmv, etc). The video will be re-packaged on the fly before being sent to the device.",
-    "DirectStreamHelp2": "Direct stream uses very little processing power with a minimal loss in video quality.",
+    "DirectStreamHelp1": "The media is in an incompatible file container (mkv, avi, wmv, etc) and the video stream is compatible with the device, but has an incompatible audio format (DTS, TRUEHD, etc) or audio channels (5.1, 7.1, etc). The video stream will be re-packaged losslessly on the fly before being sent to the device. Only audio stream will be transcoded.",
```

Comment:
```suggestion
    "DirectStreamHelp1": "The media is in an incompatible file container (MKV, AVI, WMV, etc) and the video stream is compatible with the device, but has an incompatible audio format (DTS, TRUEHD, etc) or number of audio channels. The video stream will be repackaged losslessly on the fly before being sent to the device. Only the audio stream will be transcoded.",
```

Reply:
The file container could be compatible, it's all about the audio being incompatible.